src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -774,6 +774,14 @@ public function compile(bool $resolveEnvPlaceholders = false): void
         }
     }
 
+    public function reset(): void
+    {
+        parent::reset();
+
+        $this->envPlaceholders = [];
+        $this->envCounters = [];
```

Comment:
this one should not be reset, it's a compile-time info

Reply:
done, removed it.